.coafile

```diff
@@ -1,5 +1,5 @@
 [Default]
-files = *.py, coala_langserver/*.py, tests/**/*.py
+files = *.py, coalals/**/*.py, tests/**/*.py
```

Comment:
why not **.py ?

this should be done as a separate PR, so we can see what(if any) changes are needed to make all python files comply with the rules

Reply:
I have some performance measuring scripts, env directory etc in the project directory and so might any other devs or users for this project. I think it would be good to leave it as is and not use a wider glob.